sdk/src/test/scala/io/horizen/account/websocket/WebSocketAccountServerEndpointTest.scala

```diff
@@ -106,6 +106,19 @@ class WebSocketAccountServerEndpointTest extends JUnitSuite with MockitoSugar wi
     null
   }
 
+  @Test
+  def allowedOriginsTest(): Unit = {
+    server = WebSocketAccountServerRef(mockedSidechainNodeViewHolderRef, rpcProcessor, 9035, Seq("127.0.0.2"))
+
+    val cec = ClientEndpointConfig.Builder.create.build
+    val client = ClientManager.createClient
+
+    val countDownController: CountDownLatchController = new CountDownLatchController(1)
```

Comment:
why do we need `CountDownLatchController` here?

Reply:
Because we need it to create a WsEndpoint instance, but maybe I miss something?